src/hardware/StandardLED.cpp

```diff
@@ -1,21 +1,18 @@
 /**
  * @file StandardLED.cpp
- * 
+ *
  * @brief An LED connected to a GPIO pin
  */
 
-#include "GPIOPin.h"
 #include "StandardLED.h"
+#include "GPIOPin.h"
 
-StandardLED::StandardLED(GPIOPin& gpioInstance) : gpio(gpioInstance) {}
+StandardLED::StandardLED(GPIOPin& gpioInstance)
+    : gpio(gpioInstance) {}
 
-void StandardLED::turnOn() {
-    gpio.write(HIGH);
-}
+void StandardLED::turnOn() { gpio.write(HIGH); }
 
-void StandardLED::turnOff() {
-    gpio.write(LOW);
-}
```

Comment:
this change should also not happen

Reply:
Jep, was also the "allow short functions on single line" which I now switched off.